src/routes/posts.js

```diff
@@ -33,7 +33,32 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET ALL ARTICLES (feed)
+
+const fetchPosts = async (req, res, next) => {
+    try {
+        const feed = await db.query('SELECT * FROM posts')
+        const allArticles = feed.rows
+
+        res.status(201).json({
+            status: 'success',
+            data: allArticles.map((articles) => ({
```

Comment:
it should be `article`, we map through single documents not documents. so far its one article you're dealing with, it should be `article`

Reply:
alright, i will work on it